config/helm/chart/default/templates/Common/opentelemetry-collector/clusterrole-opentelemetry-collector.yaml

```diff
@@ -15,10 +15,10 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: dynatrace-extensions-prometheus
+  name: dynatrace-opentelemetry-prometheus
```

Comment:
Only the service account shall be renamed, the other RBAC objects are still only related to extensions.

Reply:
@StefanHauth, what about folders? Do I need to keep the old folder name?

`Common/extensions-opentelemetry-collector/clusterrole-extensions-opentelemetry-collector.yaml`?

or move `clusterrole-extensions-opentelemetry-collector.yaml` to new folder but keep old content?